src/samples/flow/tilt-angle/Makefile

```diff
@@ -0,0 +1,72 @@
+ifeq (y,$(or $(BUILD_TYPE_RELEASE),$(BUILD_TYPE_DEBUG)))
+#
+# In-tree makefile
+#
+obj-$(FLOW_TILT_ANGLE_ARCTANGENT_NODE) += arctangent.mod
+obj-arctangent-$(FLOW_TILT_ANGLE_ARCTANGENT_NODE) := arctan-spec.json
+obj-arctangent-$(FLOW_TILT_ANGLE_ARCTANGENT_NODE) += arctan-node.o
+obj-arctangent-$(FLOW_TILT_ANGLE_ARCTANGENT_NODE)-type := flow
+obj-arctangent-$(FLOW_TILT_ANGLE_ARCTANGENT_NODE)-extra-ldflags += -lm
+
+sample-$(FLOW_TILT_ANGLE_SAMPLE) += tilt_angle
+sample-tilt_angle-$(FLOW_TILT_ANGLE_SAMPLE) := tilt_angle.fbp
+else
+#
+# Out-of-tree makefile
+#
+## variables share with external
+CC ?= gcc
+SYSROOT ?= /
+LIBDIR ?= /usr/lib/
+DESTDIR ?= $(SYSROOT)
+SOLINSTALLEDPATH ?= $(SYSROOT)
+## variables use locally
+DESTDIRNOBACKSLASH = $(patsubst %/,%,$(DESTDIR))
+LIBDIRNOFRONTSLASH = $(patsubst /%,%,$(LIBDIR))
+DESTLIBPATHNOBACKSLASH = $(patsubst %/,%,$(DESTDIRNOBACKSLASH)/$(LIBDIRNOFRONTSLASH))
+SOLINSTALLEDPATHNOBACKSLASH = $(patsubst %/,%,$(SOLINSTALLEDPATH))
+SOLLIBPATH = $(SOLINSTALLEDPATHNOBACKSLASH)/$(LIBDIRNOFRONTSLASH)
+SOLBINPATH = $(SOLINSTALLEDPATHNOBACKSLASH)/usr/bin/
+GENSCHEMA = $(SOLINSTALLEDPATHNOBACKSLASH)/usr/share/soletta/flow/schemas/node-type-genspec.schema
+## compiler flags
+EXTRA_CFLAGS += -fstack-protector-strong -fPIE -fPIC -O2 -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security
+EXTRA_LDFLAGS += -z noexecstack -z relro -z now -pie
+CFLAGS += --sysroot=$(SYSROOT) -I$(SOLINSTALLEDPATHNOBACKSLASH)/usr/include/soletta/ -DSOL_FLOW_NODE_TYPE_MODULE_EXTERNAL=1 $(EXTRA_CFLAGS)
+LDFLAGS += -L$(SOLLIBPATH) -lsoletta -lm $(EXTRA_LDFLAGS)
```

Comment:
Those variables should come from `pkg-config`. Soletta installs a .pc that contains all these information, we should use it, otherwise if solettas were installed in a different path it will not work. Right ?